charts/vela-core/templates/defwithtemplate/task.yaml

```diff
@@ -36,6 +37,8 @@ spec:
       	// +short=i
       	image: string
       
+        // +usage=Define job restartpolicy policy, the value should be Never or OnFailur,,is not Always
+        policy: *"Never"|string
```

Comment:
Let's name this parameter 'restart'.

Similar to: https://docs.docker.com/config/containers/start-containers-automatically/

Reply:
I feel that following the naming in K8S can make it easier to understand this parameter

Reply:
On the contrary, 'policy' doesn't make any sense from user's perspective. It could be rollout policy, update policy etc.

We either name it as 'restart' or 'restartPolicy' for user parameter.

Reply:
Yes, restartpolicy seems more appropriate as a parameter

Reply:
I'd suggest `restart` for short ^^

Reply:
Ok, long parameters are a bit cumbersome